runtime/benches/bank.rs

```diff
@@ -4,73 +4,160 @@ extern crate test;
 
 use solana_runtime::bank::*;
 use solana_runtime::bank_client::BankClient;
+use solana_runtime::loader_utils::{create_invoke_instruction, load_program};
+use solana_sdk::account::KeyedAccount;
 use solana_sdk::client::AsyncClient;
+use solana_sdk::client::SyncClient;
 use solana_sdk::genesis_block::GenesisBlock;
-use solana_sdk::hash::hash;
+use solana_sdk::instruction::InstructionError;
+use solana_sdk::native_loader;
+use solana_sdk::pubkey::Pubkey;
 use solana_sdk::signature::{Keypair, KeypairUtil};
-use solana_sdk::system_transaction;
-use solana_sdk::timing::{DEFAULT_TICKS_PER_SLOT, MAX_RECENT_BLOCKHASHES};
 use solana_sdk::transaction::Transaction;
+use std::sync::Arc;
+use std::thread::sleep;
+use std::time::Duration;
 use test::Bencher;
 
-// Create transactions between unrelated parties.
-pub fn create_sample_transactions(bank: &Bank, mint_keypair: &Keypair) -> Vec<Transaction> {
+const BUILTIN_PROGRAM_ID: [u8; 32] = [
```

Comment:
what's this?

Reply:
Test instruction processor like `system` but does nothing but call the local `process_instruction`

Reply:
But where'd that magic number of 134 come from?

Reply:
Random, how about the ascii representation of `BUILTIN_PROGRAM_ID`? :-) . Did you have a specific concern that I should address?

Reply:
Just trying to figure out what it's going to conflict with